framework/transport/ieee1905_transport/ieee1905_transport_packet_processing.cpp

```diff
@@ -577,6 +579,64 @@ bool Ieee1905Transport::forward_packet(Packet &packet)
     return true;
 }
 
+/**
```

Comment:
(nitpick) doxygen comment should be in the header.

Reply:
I agree, but since currently the transport doesn't follow this guideline I prefer to align to what is used in the transport. The whole thing is not doxygen compliant, we should fix this --> https://github.com/prplfoundation/prplMesh/issues/1322